webserver/views/api/v1/test/test_core.py

```diff
@@ -35,6 +35,10 @@ def test_get_low_level(self):
 
         resp = self.client.get("/api/v1/%s/low-level" % mbid)
         self.assertEqual(resp.status_code, 200)
+        self.assertIn('X-RateLimit-Limit', resp.headers)
```

Comment:
are we able to test specific values? or is that functionality already tested in BU?

Reply:
Tested here: https://github.com/metabrainz/brainzutils-python/commit/3738c96d669e678e2ab5b05faa7b28c9526f8d11#diff-3f0b606a8b8d994781dff384af815f83R14